src/deploy/functions/runtimes/node/triggerParser.js

```diff
@@ -7,7 +7,28 @@ var EXIT = function () {
   process.exit(0);
 };
 
-(function () {
+/**
+ * Dynamically load import function to prevent TypeScript from
+ * transpiling into a require.
+ *
+ * See https://github.com/microsoft/TypeScript/issues/43329.
+ */
+// eslint-disable-next-line @typescript-eslint/no-implied-eval
+var dynamicImport = new Function("modulePath", "return import(modulePath)");
+
+function loadModule(packageDir) {
```

Comment:
Should this be an async function?

Reply:
It can be? Does making the function `async` matter in this case?